File: CC/Codigos_Fonte_Relatorio_1/micro10/micro10.pas
program micro10;

function Fatorial(numero : integer) : integer;
begin
    if(numero <= 0 ) then Fatorial := 1
    else Fatorial := numero* Fatorial(numero-1)
end;
var
    numero, fat : integer;

begin
    write('Digite um número: ');
    read(numero);

    fat := Fatorial(numero);

    write('O fatorial de ');
    write(numero);
    write(' é ');
    write(fat);
end.

File: CC/Codigos_Fonte_Relatorio_1/micro10/micro10.java

public class micro10{

    public static void main(String[] args){
        int numero, fat;

        System.out.println("Digite um numero: ");
        numero = Integer.parseInt(System.console().readLine());

        fat = fatorial(numero);
        
        System.out.println("O fatorial de "+ numero);
        System.out.println("é");
        System.out.println(fat);
    }

    public static int fatorial(int numero){
        if(numero <= 0) return 1;
        else return numero * fatorial(numero - 1);
    }
}
